configs/config.yaml

```diff
@@ -0,0 +1,42 @@
+train:
+  batch_size: 64
+  epochs: 100
+  log_interval: 10
+  lr: 1e-6
+
+logger:
+  results_path: "results/"
+  checkpoint_path: "checkpoints/"
```

Comment:
We should take care to ensure existence of this folder - you can either track some .keep file with git, or create the folder from python level before training starts